modules/installation-creating-aws-worker.adoc

```diff
@@ -0,0 +1,159 @@
+  },
+  {
+    "ParameterKey": "WorkerSubnet", <5>
+    "ParameterValue": "yes" <6>
+  },
+  {
+    "ParameterKey": "WorkerSecurityGroupId", <7>
+    "ParameterValue": "sg-042ebc4dd65c71d67" <8>
+  },
+  {
+    "ParameterKey": "IgnitionLocation", <9>
+    "ParameterValue": "https://api.<cluster_name>.<domain_name>:22623/config/master" <10>
+  },
+  {
+    "ParameterKey": "CertificateAuthorities", <11>
+    "ParameterValue": "" <12>
+  },
+  {
+    "ParameterKey": "WorkerInstanceProfileName", <13>
+    "ParameterValue": "" <14>
+  },
+  {
+    "ParameterKey": "WorkerInstanceType", <15>
+    "ParameterValue": "m4.large" <16>
+  },
+  {
+    "ParameterKey": "AutoRegisterELB", <27>
+    "ParameterValue": "yes" <28>
+  },
+  {
+    "ParameterKey": "RegisterNlbIpTargetsLambdaArn", <29>
+    "ParameterValue": "1" <30>
+  },
+  {
+    "ParameterKey": "ExternalApiTargetGroupArn", <31>
+    "ParameterValue": "1" <32>
+  },
+  {
+    "ParameterKey": "InternalApiTargetGroupArn", <33>
+    "ParameterValue": "1" <34>
+  },
+  {
+    "ParameterKey": "InternalServiceTargetGroupArn", <35>
+    "ParameterValue": "1" <36>
+  }
+]
+----
+<1> The name for your cluster infrastructure that is encoded in your Ignition
+config files for the cluster.
+<2> Specify the infrastructure name that you extracted from the Ignition config
+file metadata, which has the format `<cluster-name>-<random-string>`.
+<3> Current {op-system-first} AMI to use for the worker nodes.
+<4> Specify an `AWS::EC2::Image::Id` value.
+<5> A subnet, preferably private, to launch the worker nodes on.
+<6> Specify a subnet from the `PrivateSubnets` value from the output of the
+CloudFormation template for DNS and load balancing.
+<7> The worker security group ID to associate with worker nodes.
+<8> Specify the `WorkerSecurityGroupId` value from the output of the
+CloudFormation template for the security group and roles.
+<9> The location to fetch bootstrap Ignition config file from.
+<10> Specify the generated Ignition config location,
+`https://api.<cluster_name>.<domain_name>:22623/config/worker`.
+<11> Base64 encoded certificate authority string to use.
+<12> Specify the value from the `worker.ign` file that is in the installation
+directory. This value is the long string with the format
+`data:text/plain;charset=utf-8;base64,ABC...xYz==`.
+<13> The IAM profile to associate with worker nodes.
+<14> Specify the `WorkerInstanceProfile` parameter value from the output of
+the CloudFormation template for the security group and roles.
+<15> The type of AWS instance to use for the control plane nodes.
+<16> Allowed values:
+* `m4.large`
+* `m4.xlarge`
+* `m4.2xlarge`
+* `m4.4xlarge`
+* `m4.8xlarge`
+* `m4.10xlarge`
+* `m4.16xlarge`
+* `c4.large`
+* `c4.xlarge`
+* `c4.2xlarge`
+* `c4.4xlarge`
+* `c4.8xlarge`
+* `r4.large`
+* `r4.xlarge`
+* `r4.2xlarge`
+* `r4.4xlarge`
+* `r4.8xlarge`
+* `r4.16xlarge`
+
+. Launch the template:
++
+----
+$ aws cloudformation create-stack --stack-name <name> \ <1>
```

Comment:
The order of events is confusing here.  Can we bring the example template up above executing the `aws cloudformation create-stack ...`?   Maybe at minimum create a hyperlink to the example template?  Customers will drive serially through this procedure and execute the `aws cloudformation create-stack ...` before the `<template>.yaml` file is created.